### PR TITLE
Introduce Prettifier

### PR DESCRIPTION
**Pull request by kevgo**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: keeps source code properly formatted
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/kevgo/prettifier/tree/master/test)
- [x] [Has documentation](https://github.com/kevgo/prettifier/blob/master/README.md)
- [x] [Is open source](https://github.com/kevgo/prettifier)
- [x] [Has a license](https://github.com/kevgo/prettifier/blob/master/LICENSE)
- [x] [Has a code of conduct](https://github.com/kevgo/prettifier/blob/master/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: kevgo

-----
[View rendered _apps/prettifier.md](https://github.com/kevgo/probot.github.io/blob/patch-1/_apps/prettifier.md)

### CODE REVIEW

File: _apps/prettifier.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Prettifier
+description: Prettifier keeps your repository content formatted using [https://prettier.io](Prettier).
+slug: prettifier
+screenshots:
+- https://raw.githubusercontent.com/kevgo/prettifier/master/documentation/screenshot_annotated.png
+authors: [kevgo]
+repository: https://github.com/kevgo/prettifier
+host: https://prettifier-prod.herokuapp.com/probot
```
Comment (gr2m) [inline _apps/prettifier.md]: Can you remove the `/probot` suffix?

```suggestion
host: https://prettifier-prod.herokuapp.com
```

Comment (kevgo) [reply _apps/prettifier.md]: done

**Comment by kevgo** [comment]
Thanks for the quick review! I have adjusted the host. More config options are definitely on the roadmap. I'll add templates for custom commit messages and PR title and body to the Prettifier configuration file once there is demand for it.

**Comment by kevgo** [comment]
Btw check out how Prettifier has formatted this pull request in https://github.com/probot/probot.github.io/pull/330/commits/df42b1313f53ca4924b8787c615a0caa8fa4c439. A code formatting bot introduces itself by formatting the pull request that introduces it to the world!

**Pull request merged**